apps/frontend/client/src/lib/test_preload.ts

```
@@ -899,6 +899,138 @@ mock.module('@aikami/frontend/storage', () => ({
+          bestCount = count;
+          best = { col, row };
+        }
+      }
+    }
+    return best !== undefined && bestCount >= minContent ? best : undefined;
+  }),
+  encodeLpcPreviewState: mock((state: { layers: Array<{ slotDefIndex: number; variantIndex: number }> }) => {
+    const params = new URLSearchParams();
+    for (let i = 0; i < state.layers.length; i++) {
+      const layer = state.layers[i];
+      if (layer) params.set(`l${i}`, `${layer.slotDefIndex}:${layer.variantIndex}`);
```

Biome (2.5.7)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)